Makefile.am

```diff
@@ -127,7 +130,8 @@ DISTCHECK_CONFIGURE_FLAGS = \
 	--enable-cppcheck \
 	--enable-valgrind \
 	--enable-code-coverage \
-	--enable-functional-tests
+	--enable-functional-tests \
+	--with-systemdsystemunitdir=/tmp/cc-distcheck/systemdunitdir/
```

Comment:
Actually, I didn't really want to make it unique: it's not needed (ie. not needing several distinct copies of that data) and doing several make distcheck will not end up clobbering /tmp.